python/lsst/faro/utils/calibrated_catalog.py

```diff
@@ -0,0 +1,33 @@
+# This file is part of faro.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from lsst.afw.geom import SkyWcs
+from lsst.afw.image import PhotoCalib
+from lsst.afw.table import SourceCatalog
+
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
```

Comment:
Seeing this syntax for the first time. This is a nice solution to the packaging of catalogs with their associated astrometric and photometric calibrations